samtranslator/model/apigatewayv2.py

```diff
@@ -299,3 +304,9 @@ def _get_reauthorize_every(self):  # type: ignore[no-untyped-def]
             return None
 
         return self.identity.get("ReauthorizeEvery")
+
+    @staticmethod
+    def _get_jwt_configuration(props: Optional[Dict[str, Union[str, List[str]]]]) -> Optional[JwtConfiguration]:
+        if not props:
+            return None
+        return {k.lower(): v for k, v in props.items()}
```

Comment:
Any reference that explains why this is needed?

Reply:
The issue in PR? Isn't it enough?

Reply:
So why is this specific to `JwtConfiguration`?

Reply:
If this only applies to `JwtConfiguration` because it's inconsistent with `AWS::ApiGatewayV2::Authorizer`, we should explain that. Maybe just a comment above saying e.g. "AWS::ApiGatewayV2::Authorizer uses uppercase, OpenAPI lowercase; make lowercase to avoid confusion" or something.

Reply:
Actually I see this comment: 
Use lowercase issuer and audience properties with `AWS::Serverless::HttpApi` and uppercase Issuer and Audience properties with `AWS::ApiGatewayV2::Authorizer` here: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-httpapi-oauth2authorizer.html#sam-httpapi-oauth2authorizer-jwtconfiguration

I'm just wondering if there was a specific reason to do this.

Reply:
The reason for the comment in the documentation? Because SAM-T provided not the best UX. Because of the way an authoriser is created. I know about the comment.

Reply:
@hoffa I've explained why we need to do that in the docstring

Reply:
Ok that's clearer. And there's no use case where the capitalized version has any purpose in `AWS::Serverless::HttpApi`?